packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -25,16 +27,39 @@ export default function DataViews( {
 	paginationInfo,
 } ) {
 	const ViewComponent = view.type === 'list' ? ViewList : ViewGrid;
+	const authors = [
```

Comment:
Is there a way to avoid hardcoding this into the DataViews component, in favor of a "showFilter: true" or something in the field itself?